Kilo Class Mining Ship QOL

Erikafox opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
Minorly remaps the Kilo so it's not as bad. It now does things like use functioning airlocks, have working conveyor belts, and fly forwards instead of backwards. 
<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

<!-- Tick the box below (put an X instead of a space between the brackets) if you have tested your changes and this is ready for review. Leave unticked if you have yet to test your changes and this is not ready for review. -->

- [x] I affirm that I have tested all of my proposed changes and that any issues found during tested have been addressed.

## Why It's Good For The Game
The Kilo is less unintentionally bad.
![2022 08 26-19 58 55](https://user-images.githubusercontent.com/94164348/187011792-6f56e50a-17f9-446d-a295-f868e2f1bede.png)
![dreamseeker_peZMfNOzCr](https://user-images.githubusercontent.com/94164348/187011891-3ff207b9-bede-4d5e-bea0-b189326b76b7.png)
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog

:cl:
tweak: The Kilo Class now flies to the right. You can now experience hitting bugs with your windows in real time.
add: The Kilo Class now uses dual port airlocks and AACs. 
add: The Kilo Class now has engine shutters
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

Erikafox committed:
Maurice is a name.

Erikafox committed:
main airlock now has dp vents

Erikafox committed:
Kilo now flies forward.

Erikafox committed:
areas :)

Erikafox committed:
adds a stupid fucking cargo ramp (subject to removal if its DUMB and STUPID). No doublestack atmos machinery

Erikafox committed:
THE RAMP WAS STUPID. ITS GONE

Erikafox committed:
adds the worst room ever. The Shoilet (+ newscaster). Adds a newscaster that doesn't make you hate existence.

Erikafox committed:
Maurice is real (again)

Erikafox committed:
ship is now normal sized.

Erikafox committed:
Oh my god JC engine shutters

Engine shutters?

Erikafox committed:
Fire Alarms are the devil.

Erikafox committed:
fixes the engine shutters

Erikafox committed:
One last wallmount rename.

tiramisuapimancer commented on _maps/shuttles/shiptest/whiteship_kilo.dmm:
```diff
@@ -1040,251 +1002,226 @@
+	faction = list("neutral");
+	melee_damage_lower = 5;
+	melee_damage_upper = 10;
+	name = "Maurice"
```
```suggestion
	name = "maurice"
```
Please keep his name lowercase, this was an intentional design choice when I added him

Erikafox replied:
Alrighty.

Erikafox committed:
should appease linters.

Erikafox committed:
Maurice is no longer real

Erikafox committed:
Merge branch 'master' into kilo-qol

Erikafox committed:
fixes assorted airpipe and cable related fuckups

Erikafox committed:
I saw another airpipe

Erikafox committed:
Merge branch 'kilo-qol' of https://github.com/Bjarl/Shiptest into kilo-qol

Erikafox committed:
should be the last of the doubled up items

Erikafox committed:
GIVE ME MY GREEN CHECKMARK GITHUB

Erikafox committed:
Merge branch 'master' into kilo-qol

Erikafox committed:
Merge branch 'master' into kilo-qol

Latentish reviewed (changes requested):
To start this is not a valid air cycler. While the doors do actually swap and automatically start cycles, it doesn't have enough air to refill. Moreover airlocks like this require the "high volume" versions of the dualport airvents (otherwise it takes a million years). Just one should do it for this.
The pipes ending this way is also extremely poor practice. If you're going to have the scrubbers network and the dual ports both shoot gas into space, you need to put injectors at the end.
I'd actually like to see you put at least some form of either air recycling or putting a dedicated air supply explicitly for this airlock. Otherwise it just doesn't work.
![image](https://user-images.githubusercontent.com/55075747/188242010-945d39e4-39a1-4e95-9881-5bdb39e31ad7.png)

![image](https://user-images.githubusercontent.com/55075747/188242452-fb175f9d-7edc-4e0d-8884-f8f2ecf22655.png)
You also put an o2 canister connected to the vents network here instead of an air canister. o2 can be toxic as it's just pure oxygen. 

Did you mean for the captain's room door to be bolted? This ship looks like a sty so I assume so.

7. While the kilo doesn't have one to begin with, it wouldn't be bad to give the dedicated mining ship a mining equipment vendor. (make sure to put a real one not a refill and circuit as refills are getting scrapped at some point.)

Erikafox committed:
fixes the firelocks that were apparently under all the windows and makes the air less bad.

To-Do
Remove Kilo Class

Erikafox committed:
Merge branch 'master' into kilo-qol

Erikafox commented:
I'm putting an air recycler in, putting a gas pump on the injector to space there is, and a to do note to delete this ship in my documents folder. 
>The pipes ending this way is also extremely poor practice. If you're going to have the scrubbers network and the dual ports both shoot gas into space, you need to put injectors at the end.

They go to a layer adapter that formerly went straight to an injector.
>Did you mean for the captain's room door to be bolted? This ship looks like a sty so I assume so.

It's kilo soul.

Erikafox committed:
removes the air recycler.

Erikafox committed:
Merge branch 'kilo-qol' of https://github.com/Bjarl/Shiptest into kilo-qol

Erikafox commented:
![Discord_8R22bkRW0j](https://user-images.githubusercontent.com/94164348/188256705-ca70c473-4e63-43bb-b905-4263faa59ba4.png)
I agree with Apogee. It's far too small to have one that would work, and I'd prefer to not remodel it for a qol passover.

Erikafox committed:
feex.

Erikafox committed:
puts some changes in based on feedback

Kitchen is now its own area. Status displays are a different type, cable is fixed.

ghost commented:
one floater fire alarm on the south suit storage

Erikafox committed:
fixes a fire alarm

Erikafox commented:
fixed, thanks.

Erikafox committed:
Merge branch 'master' into kilo-qol

Erikafox committed:
Merge branch 'master' into kilo-qol

Erikafox committed:
Merge branch 'master' into kilo-qol

Latentish commented:
They call me maurice.
Marg??????

Erikafox committed:
Merge branch 'master' into kilo-qol

Erikafox committed:
fixes the funny floating light I saw playing today
